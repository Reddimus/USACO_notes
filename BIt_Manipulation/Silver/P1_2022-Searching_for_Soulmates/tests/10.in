10
3934197224 1922655988
728210629 3159823609
2665564525 1467047568
2904108050 886753723
1156935423 910958158
3225258375 807851426
4159712989 3028865292
1348911999 3414018738
1135606044 3040464712
1669375703 1634681922
