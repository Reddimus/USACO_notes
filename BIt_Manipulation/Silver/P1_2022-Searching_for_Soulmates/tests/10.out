86
78
80
88
77
63
81
85
80
87
